src/scripts/seed/coach/seed.py

```
@@ -0,0 +1,379 @@
+    row = conn.execute(text("SELECT id FROM admin ORDER BY id LIMIT 1")).first()
+    if row:
+        return row[0]
+
+    now = datetime.now(timezone.utc)
+    return conn.execute(text("INSERT INTO admin (last_updated) VALUES (:last_updated) RETURNING id"), {"last_updated": now}).scalar()
+
+
+def get_or_create_account(conn: Any, account_data: dict) -> tuple[int, bool]:
+    row = conn.execute(text("SELECT id, client_id, coach_id FROM account WHERE email = :email"), {"email": account_data["email"]}).first()
+    if row:
+        account_id, client_id, coach_id = row
```

Ruff (0.15.12)

[warning] Unpacked variable `client_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)